README.md

```diff
@@ -19,6 +19,21 @@ If this works, then you will need to look for where that "GITHUB_PAT" variable i
 
 ## Using this package in a DfE data dashboard
 
+### Adding analytics to your dashboard
+
+For analytics to function on your dashboard, you will need to:
+
+- request a Google Analytics key from the [Statistics Development Team](mailto:statistics.development@education.gov.uk)
+- create a html file with the javascript required for your dashboard to connect to Google Analytics
+
+To create the latter, we provide the function `dfeshiny::initialise_analytics()`. You should run this code from the R console providing your Google Analytics code as follows:
+
+```
+initialise_analytics("ABCDE12345")
```

Comment:
I think it could be worth making it clearer (either here or in the sentence above) that the ABCDE12345 is just an example and you'd put the key we give them into this

Reply:
Done